File: Exercícios/Python Mundo 1/Desafio_15.py
# Exercício Python 15: Escreva um programa que pergunte a quantidade de Km percorridos por um carro alugado e a 
# quantidade de dias pelos quais ele foi alugado. Calcule o preço a pagar, sabendo que o carro custa R$60 por dia e R$0,15 por Km rodado.

print('=' * 6 + ' DESAFIO 15 ' + '=' * 6)

dias = int(input('[?] - Quantos dias alugados? '))
km = float(input('[?] - Quantos Km rodados? '))
preco = (dias * 60) + (km * 0.15)

print(f'[!] - O total a pagar é de R${preco:.2f}')

File: Exercícios/Scripts_C#/ScriptsUsandoC#/ScriptsUsandoC#/Desafio_15.cs
class Desafio_15
{
    public static void CalculoDeCobranca()
    {
        Console.Write("INFORME A QUANTIDADE DE [DIAS] ALUGADOS: ");
        int dias = int.Parse(Console.ReadLine().Trim());
        Console.Write("INFORME A QUANTIDADE DE [KM] RODADOS: ");
        float km = float.Parse(Console.ReadLine().Trim().Replace(".", ","));

        float preco = (60f * dias) + (km * 0.15f);
        Console.Write($"\n[DIAS ALUGADOS] - {dias}\n[KM PERCORRIDOS] - {km}\n[VALOR TOTAL] - {preco.ToString("C")}");

    }
}
